### DISCUSSION TITLE
In some cases the search is case-sensitive

### DISCUSSION THREAD

**Discussion by Hannoma** [laravel/nova-issues#5115] [Q&A] [answered]
I have defined the resource fields as searchable (with the `$search` array). When I search, the search seems to be case-sensitive. This does not happen for other resources, where the search is case-insensitive

**Comment by Hannoma** [comment, marked as answer]
Fixed this issue by changing the encoding of the underlying field in the database from `utf8mb4_bin` to `utf8mb4_unicode_ci`

**Comment by johnpuddephatt** [comment]
Searching within a Tag field is case sensitive which feels odd.

**Reply by crynobone** [reply]
If you set the database charset to use case sensitive, it would be odd to now want search to be case insensitive.

**Reply by johnpuddephatt** [reply]
I haven't "set the database to use case sensitive" – I'm searching a JSON column which I believe is always case sensitive?

The reason it's a JSON column is there are multilingual values stored in the column (I'm using spatie/laravel-tags).
